src/stdatamodels/jwst/datamodels/schemas/specwcs_miri_lrs.schema.yaml

```diff
@@ -0,0 +1,109 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/asdf/asdf-schema-1.1.0"
+id: "http://stsci.edu/schemas/jwst_datamodel/specwcs_miri_lrs.schema"
+title: MIRI LRS Spec Schema
+allOf:
+- $ref: referencefile.schema
+- $ref: keyword_pexptype.schema
+- $ref: keyword_exptype.schema
+- $ref: keyword_readpatt.schema
+- $ref: keyword_filter.schema
+- $ref: keyword_band.schema
+- $ref: subarray.schema
+- type: object
+  properties:
+    wave_table:
+      title: Wavelengths and x, y locations of wavelengths
+      fits_hdu: WAVETABLE
+      datatype:
+      - name: x_center
+        datatype: float32
+        unit: pixels
+      - name: y_center
+        datatype: float32
+        unit: pixels
+      - name: wavelength
+        datatype: float32
+        unit: microns
+      - name: x0
+        datatype: float32
+        unit: pixels
+      - name: y0
+        datatype: float32
+        unit: pixels
+      - name: x1
+        datatype: float32
+        unit: pixels
+      - name: y1
+        datatype: float32
+        unit: pixels
+      - name: x2
+        datatype: float32
+        unit: pixels
+      - name: y2
+        datatype: float32
+        unit: pixels
+      - name: x3
+        datatype: float32
+        unit: pixels
+      - name: y3
+        datatype: float32
+        unit: pixels
```

Comment:
It'd be helpful to add description (or title) for each of these fields. These can come as a minimum from the names in the FITS table.

Reply:
@nden The names in the fits file are exactly these names and no more information in fits file is provided. I will give a shot at a description and run that by Greg Sloan who I think made these reference files.

Reply:
Yes, you are right. The names are the same. I sent you a document with the description. I think it's good to add the descriptions because the meaning of the names is not obvious.

Reply:
done